example/src/main/scala/example/SignCookies.scala

```diff
@@ -0,0 +1,24 @@
+package example
+
+import zhttp.http.Cookie.{httpOnly, maxAge, path, sign}
+import zhttp.http.{Cookie, Method, Response, _}
+import zhttp.service.Server
+import zio.duration.durationInt
+import zio.{App, ExitCode, URIO}
+
+/**
+ * Example to make app using signed-cookies
+ */
+object SignCookies extends App {
+
+  // Setting cookies with an expiry of 5 days
+  private val cookie = Cookie("key", "hello") @@ maxAge(5 days)
+
+  private val app = Http.collect[Request] { case Method.GET -> !! / "cookie" =>
+    Response.ok.addCookie(cookie @@ path(!! / "cookie") @@ httpOnly @@ sign("secret") withContent "world")
```

Comment:
https://github.com/dream11/zio-http/pull/533/files#r773066949

How about this approach?